src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/LASSystem.java

```diff
@@ -15,46 +15,16 @@
  */
 package org.terasology.ligthandshadow.componentsystem.controllers;
 
-import org.terasology.engine.entitySystem.entity.EntityManager;
-import org.terasology.engine.entitySystem.entity.EntityRef;
-import org.terasology.engine.entitySystem.event.ReceiveEvent;
 import org.terasology.engine.entitySystem.systems.BaseComponentSystem;
 import org.terasology.engine.entitySystem.systems.RegisterSystem;
-import org.terasology.module.inventory.components.InventoryComponent;
-import org.terasology.module.inventory.systems.InventoryManager;
-import org.terasology.module.inventory.events.RemoveItemAction;
-import org.terasology.engine.logic.players.event.OnPlayerSpawnedEvent;
 import org.terasology.engine.registry.In;
-import org.terasology.engine.world.WorldProvider;
-import org.terasology.engine.world.block.BlockManager;
-import org.terasology.engine.world.block.items.BlockItemFactory;
 import org.terasology.engine.world.sun.CelestialSystem;
 
 @RegisterSystem
 public class LASSystem extends BaseComponentSystem {
-    @In
-    private EntityManager entityManager;
-    @In
-    private InventoryManager inventoryManager;
-    @In
-    private WorldProvider worldProvider;
-    @In
-    private BlockManager blockManager;
     @In
     private CelestialSystem celestialSystem;
 
-    /**
-     * Gives player inventory items on game start
-     */
-    @ReceiveEvent
-    public void onPlayerSpawn(OnPlayerSpawnedEvent event, EntityRef player, InventoryComponent inventory) {
-        BlockItemFactory blockFactory = new BlockItemFactory(entityManager);
-        for (int i = 0; i < inventoryManager.getNumSlots(player); i++) {
-            EntityRef itemInSlot = inventoryManager.getItemInSlot(player, i);
-            player.send(new RemoveItemAction(player, itemInSlot, true));
-        }
-    }
```

Comment:
The main reason I am removing this is because, this doesn't allow us to choose our starting inventory based on the defined prefab. All it does is remove all the items in the players inventory which doesn't give the player anything to fight with. By removing this we can settle on a more well defined starting inventory via the prefab. I will shift this to another PR.

Reply:
I think the intention behind this code was for the players to not have any items in their inventory in the lobby stage. Generally I do think that makes sense as the actual game takes place in the arena and not in the lobby.

An alternative for the lobby stage might be to make players invincible. Thus, players could already test out the items without being able to kill each other at that stage yet. However, for placeable (traps) / consumable (ammunition) items, we would need to ensure they still start into the game with the full set.

Shifting this to another PR makes sense, so we can discuss this there in more detail :+1: